iOS-13-Migration-Guide.md

```diff
@@ -5,6 +5,10 @@ iOS 13 introduced changes in push notifications handling. This document describe
 - [iOS 13 & Xcode 10 or below](#ios-13--xcode-10-or-below)
 - [iOS 13 & Xcode 11](#ios-13--xcode-11)
 
+## Register Your App with PushKit at Launch Time
```

Comment:
I don't think it makes sense here either. It should be a step under each migration guide.

Reply:
@idelgado addressed the review feedbacks. lmk